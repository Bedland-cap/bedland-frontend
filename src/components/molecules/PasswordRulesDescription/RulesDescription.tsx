import { RulesDescriptionContainer, OrderedRulesDesc } from './RulesDescription.styled';

const RulesDescription = () => (
  <RulesDescriptionContainer>
    <span> Remember, your password must include:</span>{' '}
    <OrderedRulesDesc style={{ listStyleType: 'disc' }}>
      <li>8-24 characters</li>
      <li>At least one uppercase character</li>
      <li>At least one special character</li>
    </OrderedRulesDesc>
  </RulesDescriptionContainer>
);
export default RulesDescription;
