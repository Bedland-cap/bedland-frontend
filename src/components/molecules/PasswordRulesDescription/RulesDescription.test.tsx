import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import RulesDescription from './RulesDescription';

describe('RulesDescription', () => {
  it('renders correctly', () => {
    const snapshot = renderer
      .create(
        <BrowserRouter>
          <RulesDescription />
        </BrowserRouter>,
      )
      .toJSON();
    expect(snapshot).toMatchSnapshot();
  });
});
