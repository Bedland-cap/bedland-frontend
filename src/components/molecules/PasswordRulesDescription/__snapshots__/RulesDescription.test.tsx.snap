// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RulesDescription renders correctly 1`] = `
<div
  className="sc-bcXHqe frrnbC"
>
  <span>
     Remember, your password must include:
  </span>
   
  <ol
    className="sc-gswNZR dnILCN"
    style={
      {
        "listStyleType": "disc",
      }
    }
  >
    <li>
      8-24 characters
    </li>
    <li>
      At least one uppercase character
    </li>
    <li>
      At least one special character
    </li>
  </ol>
</div>
`;
